Assets/Plugins/CommentToTooltip/TooltipGenerator.cs

```diff
@@ -50,33 +56,58 @@ public TooltipGenerator()
             field => field declaration
             */
 
+            
             const string nonTooltipAttributes =
                 @"(?<attributes>^[ \t]*\[(?!([ \t]*(?:UnityEngine.)?Tooltip))[^]]+\]\s*(?=^))*";
             const string nonCommentRegexPart =
-                @"\s*(?=^)" + nonTooltipAttributes + @"(?<tooltip>^[ \t]*\[(?:UnityEngine.)?Tooltip\(""(?<tooltipContent>[^""]*)""\)\]\s*(?=^))?" + nonTooltipAttributes + @"(?<field>(?<beginning>^[ \t]*)public\s+[^\s;=]+\s+[^\s;=]+\s*(?>=[^;]+)?;)";
+                @"\s*(?=^)" + 
+                nonTooltipAttributes + 
+                @"(?<tooltip>^[ \t]*\[(?:UnityEngine.)?Tooltip\(""(?<tooltipContent>[^""]*)""\)\]\s*(?=^))?"
+                + nonTooltipAttributes + 
+                @"(?<field>(?<beginning>^[ \t]*)public\s+[^\s;=]+\s+[^\s;=\\(]+\s*(?>=[^;]+)?;)";

```

Comment:
this line is the one that fixes the bug that causes explicitly public interface methods to get tooltipped.